src/bootstrap/check.rs

```diff
@@ -461,6 +461,48 @@ impl Step for RustdocJS {
     }
 }
 
+#[derive(Debug, Copy, Clone, Hash, PartialEq, Eq)]
+pub struct RustdocTheme {
+    pub compiler: Compiler,
+    pub host: Interned<String>,
+}
+
+impl Step for RustdocTheme {
+    type Output = ();
+    const DEFAULT: bool = true;
+    const ONLY_HOSTS: bool = true;
+
+    fn should_run(run: ShouldRun) -> ShouldRun {
+        run.path("src/tools/rustdoc-themes")
+    }
+
+    fn make_run(run: RunConfig) {
+        let compiler = run.builder.compiler(run.builder.top_stage, run.host);
+
+        run.builder.ensure(RustdocTheme {
+            compiler: compiler,
+            host: run.builder.build.build,
+        });
+    }
+
+    fn run(self, builder: &Builder) {
+        let rustdoc = builder.out.join("bootstrap/debug/rustdoc");
```

Comment:
This should be a call to `builder.rustdoc`. Otherwise it may not have been built yet.